### DISCUSSION TITLE
I want PHP bash only for admins

### DISCUSSION THREAD

**Discussion by YeesterPlus** [community/community#127018] [Programming Help]
### Select Topic Area

Question

### Body

How do I add php without downloads

**Comment by rohispace90** [comment]
Hello dear;
Adding PHP to your workflow without downloading and installing it directly on your machine can be done using various online platforms and services.

Here are a few methods to use PHP online:

1- Online PHP Editors/Compilers.
2- Cloud-Based IDEs.
3- Using PHP on Web Hosting Services.
4- Using Containers:
You can run PHP in a containerized environment using Docker without installing PHP directly on your local machine. This requires installing Docker, but once set up, you can use pre-configured PHP containers.

Example command to run PHP in a Docker container:
`docker run -it --rm php:latest`

**Reply by ebndev** [reply]
@rohispace90 thanks for participating in Discussions! Glad to see you trying to help others resolve their issues.

@YeesterPlus let us know if the response above was helpful 😄
